Makefile

```
@@ -1,16 +1,117 @@
+.PHONY: build install generate docker-build docker-run test run run-debug
+
+$(TEMPDIR)/:
+	mkdir -p $@
+
+$(BUILDDIR)/:
+	mkdir -p $@
+
+# ------------------------------------------------------------------------------
+
+# Download wasmvm static lib if missing
+wasmvmlib: $(TEMPDIR)/
```

checkmake (0.2.2)

[warning] Target body for "wasmvmlib" exceeds allowed length of 5 (15).

(maxbodylength)